composer.json

```
@@ -77,5 +77,59 @@
             "ergebnis/composer-normalize": true
         },
         "sort-packages": true
+    },
+    "scripts": {
+        "cs:check": "php php-cs-fixer fix --dry-run --diff",
+        "cs:fix": "php php-cs-fixer fix",
+        "dev-tools:check": [
+            "./dev-tools/check_file_permissions.sh",
+            "./dev-tools/check_trailing_spaces.sh"
+        ],
+        "dev-tools:docs": "php dev-tools/doc.php",
+        "dev-tools:install": "composer -d dev-tools install",
```

```suggestion
        "dev-tools:install": "@composer -d dev-tools install",
```
see https://getcomposer.org/doc/articles/scripts.md#calling-composer-commands